main/src/library/File.flix

```diff
@@ -423,7 +460,7 @@ namespace File {
             close(w);
 
             if (checkError(w)) {
-                Err("An error occurred when writing lines to the file: "+f)
+                Err("An error occurred when writing lines to the file: " + f)
```

Comment:
I have a slight preference for writing this instead:

Err("An error occurred when writing lines to the file: '${f}'.")

Reply:
If you fix this, then I can merge :)

Reply:
Should be ready now :)